third_party/arrow.BUILD

```diff
@@ -25,19 +26,29 @@ flatbuffer_cc_library(
     out_prefix = "cpp/src/arrow/ipc/",
 )
 
+genrule(
+  name = "arrow_util_config",
+  srcs = ["cpp/src/arrow/util/config.h.cmake"],
+  outs = ["cpp/src/arrow/util/config.h"],
+  cmd =  "sed 's/@ARROW_VERSION_MAJOR@/0/g ; s/@ARROW_VERSION_MINOR@/14/g ; s/@ARROW_VERSION_PATCH@/1/g ; s/#cmakedefine/#undef/g' $< > $@",
```

Comment:
This is pretty hacky, but it's the only way I could figure out how to do it. Any other ideas @yongtang ?

Reply:
@BryanCutler The Bazel allows using Starlark for complex logic. Starlark essentially is a dialect of python. However, I am afraid using Starlark to achieve `cmake` functionality will be more complicated than `sed`.

Reply:
I see, well since this is a simple replace. It's probably not worth doing anything complicated. Is there an easy way to parametrize the version here?